### PR TITLE
Create v-vvv-e-eee

### PR DESCRIPTION
**Pull request by harryharrold**
Fixes #

## Description

Adds new blog post!

(Probably)

-
-

### CODE REVIEW

File: src/data/blog/v-vvv-e-eee.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+amp: null
+author_name: 'Harry Harrold'
+author_slug: 'harry-harrold'
+author_image: '//www.gravatar.com/avatar/febf032ab16fef166824f5172cd87393?s=250&d=mm&r=x'
+author_cover: null
+author_facebook: null
+author_twitter: null
+---
+
+Every sprint, Neontribe’s developers are faced with a stack of work, in an order. Every sprint, we start with work from the top of the stack, and carry on for as long as we have planned
```
Comment (katjam) [inline src/data/blog/v-vvv-e-eee.md]: .

File: src/data/blog/v-vvv-e-eee.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+We order this stack using Vs and Es; rough of the Value and Effort of a piece of work. Normally, these pieces of work at least start as user stories; value we are delivering to a user of the software we’re creating. Every piece of work will start with one to three Vs and one to three Es.
```
Comment (katjam) [inline src/data/blog/v-vvv-e-eee.md]: `; rough of the Value` is missing some words I think.

File: src/data/blog/v-vvv-e-eee.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+It *must not* be the case that all user stories are deemed to have the same Value and the same Effort. The reason we do any of this is to be able to prioritise work for a sprint which we’ll get through in a ranked order. As developers, we don’t mind what order we end up with, as long as the team understands the tradeoffs. That makes ranking a series of difficult decisions, and they end up with the product owner. We do start with a default though, just to have something to work on. We start with highest Value, lowest Effort and proceed from there.
+
+You’ll have seen by now we really have no such thing as "Must Haves", just a prioritised list. At any time, a Product Owner can re-order the work in a sprint, and the only opportunity cost is the work already done on de-prioritised work. They're in control, they decide what's most important at any time.
```
Comment (charlesstrange2) [inline src/data/blog/v-vvv-e-eee.md]: Small caps on `Product Owner`, as neither product owner or developer are caps in the rest of the peice.

File: src/data/blog/v-vvv-e-eee.md (hunk 1 above)
Comment (harryharrold) [reply src/data/blog/v-vvv-e-eee.md]: "Every sprint, we start with work from the top of the stack, and carry on until the end of what's usually a fortnight of solid coding." reads better - thank you.

File: src/data/blog/v-vvv-e-eee.md (hunk 2 above)
Comment (harryharrold) [reply src/data/blog/v-vvv-e-eee.md]: Thank you! Replacing with "rough ideas of the Value and Effort of a piece of work."

File: src/data/blog/v-vvv-e-eee.md (hunk 3 above)
Comment (harryharrold) [reply src/data/blog/v-vvv-e-eee.md]: product owners un-capitalised.

Thank you.

**Comment by harryharrold** [comment]
Thanks @charlesstrange2  and @katjam !

File: src/data/blog/v-vvv-e-eee.md (hunk 1 above)
Comment (katjam) [reply src/data/blog/v-vvv-e-eee.md]: Still missing a fullstop after 'planned'

File: src/data/blog/v-vvv-e-eee.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
+---
+title: 'Value and effort and prioritisation.'
```
Comment (Nikomus) [inline src/data/blog/v-vvv-e-eee.md]: I don't think we need a full stop in the title. I'd say use title case but we haven't consistently used that before..

**Comment by charlesstrange2** [comment]
@harryharrold i cant see any changes you made since the last review?

File: src/data/blog/v-vvv-e-eee.md (hunk 4 above)
Comment (harryharrold) [reply src/data/blog/v-vvv-e-eee.md]: Thank you!

File: src/data/blog/v-vvv-e-eee.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+(VVVV) is not an acceptable value; it probably means there’s something epic there that needs to be shattered into smaller tasks.
+
+These assessments of Value are used to help prioritise user stories. The more Valuable a user story is, the more likely it is to be included in the software the project eventually delivers but it’s the product owner’s call. Value gives us a good first stab at prioritisation, which we then consider again when we know more about Effort.
+
+(E) means a user story will take some Effort. Any user story needs design thinking, coding, testing and some discussions to deliver. No user story, however seemingly trivial, is without effort to finish to a satisfactory level of quality. A developer who gives a story an E probably knows exactly how to do it and that it’ll be only a few lines of code. 
+
+(EE) means a user story will take Extra Effort. It's more work than a user story that which only takes Effort, but less than one which takes Extreme effort. It’s likely a developer who gives a story an EE will have a pretty good idea how to do it but it's more than a few lines of code.
```
Comment (Nikomus) [inline src/data/blog/v-vvv-e-eee.md]: "that which" probably just that?

File: src/data/blog/v-vvv-e-eee.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+amp: null
+author_name: 'Harry Harrold'
+author_slug: 'harry-harrold'
+author_image: '//www.gravatar.com/avatar/febf032ab16fef166824f5172cd87393?s=250&d=mm&r=x'
+author_cover: null
+author_facebook: null
+author_twitter: null
+---
+
+Every sprint, Neontribe’s developers are faced with a stack of work, in an order. Every sprint, we start with work from the top of the stack, and carry on until the sprint finishes.
```
Comment (Nikomus) [inline src/data/blog/v-vvv-e-eee.md]: Should "Every sprint" be repeated in both sentences?

Comment (harryharrold) [reply src/data/blog/v-vvv-e-eee.md]: Yup. Well, it was intentionally done to demonstrate a considered process.

File: src/data/blog/v-vvv-e-eee.md (hunk 5 above)
Comment (harryharrold) [reply src/data/blog/v-vvv-e-eee.md]: Good spot- thank you.

File: src/data/blog/v-vvv-e-eee.md (hunk 1 above)
Comment (harryharrold) [reply src/data/blog/v-vvv-e-eee.md]: ...sorry, I can't see where? My re-write took the word planned out, I think?

File: src/data/blog/v-vvv-e-eee.md (hunk 2 above)
Comment (katjam) [reply src/data/blog/v-vvv-e-eee.md]: Still says "rough of the Value and Effort"

**Comment by harryharrold** [comment]
> @harryharrold i cant see any changes you made since the last review?

@charlesstrange2 - could you check for me? I think that change has been made?

File: src/data/blog/v-vvv-e-eee.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+
+(VVVV) is not an acceptable value; it probably means there’s something epic there that needs to be shattered into smaller tasks.
+
+These assessments of Value are used to help prioritise user stories. The more Valuable a user story is, the more likely it is to be included in the software the project eventually delivers but it’s the product owner’s call. Value gives us a good first stab at prioritisation, which we then consider again when we know more about Effort.
+
+(E) means a user story will take some Effort. Any user story needs design thinking, coding, testing and some discussions to deliver. No user story, however seemingly trivial, is without effort to finish to a satisfactory level of quality. A developer who gives a story an E probably knows exactly how to do it and that it’ll be only a few lines of code. 
+
+(EE) means a user story will take Extra Effort. It's more work than a user story that only takes Effort, but less than one which takes Extreme effort. It’s likely a developer who gives a story an EE will have a pretty good idea how to do it but it's more than a few lines of code.
```
Comment (charlesstrange2) [inline src/data/blog/v-vvv-e-eee.md]: No reference to Extreme effort, the next one is Extra Extra Effort

Comment (harryharrold) [reply src/data/blog/v-vvv-e-eee.md]: Extreme removed.

**Pull request merged**